packages/framer-motion/src/render/utils/animation-state.ts

```diff
@@ -384,7 +384,13 @@ export function createAnimationState(
 
         state[type].isActive = isActive
 
-        return animateChanges(options, type)
+        const animations = animateChanges(options, type)
+
+        for (const key in state) {
+            state[key].protectedKeys = {}
```

Comment:
😄

Reply:
Does this mean animations triggered by setActive will never be protected if the value is unchanged? Would this cause other issues?

Reply:
None that have been caught by tests 😅

This shouldnt be a problem because setActive will start the animation from the instigating component down through the tree whereas animations fired from useEffect will go from components bottom-up, and this is when we need to protect animations from being overridden by parents

Reply:
ah nice context, thanks for confirming!